draft-ietf-quic-recovery.md

```diff
@@ -100,14 +100,15 @@ interpretation of TCP loss detection mechanisms.
 Every packet may contain several frames.  We outline the frames that are
 important to the loss detection and congestion control machinery below.
 
-* Retransmittable frames are frames requiring reliable delivery.  The most
-  common are STREAM frames, which typically contain application data.
+* Congestion controlled frames are frames that count towards bytes in
+  flight and need acknowledgement.  The most common are STREAM frames,
+  which typically contain application data.
 
 * Crypto handshake data is sent on stream 0, and uses the reliability
   machinery of QUIC underneath.
 
 * ACK frames contain acknowledgment information.  ACK frames contain one or more
-  ranges of acknowledged packets.
+  ranges of acknowledged packets and are not congestion controlled.
```

Comment:
This now sounds like ACKs never count towards the bytes_in_flight. As far as I understand the rest of this PR, this only applies for ACK-only packets though, right?